setup.py

```diff
@@ -71,6 +71,7 @@ def build_extension(self, ext):
     'webob',
     'pytz',
     'redis',
+    'requests',
```

Comment:
can you add this to test_requirements/requirements-base.txt as well? this ensures that it is added to the pip cache on travis.